modules/bibformat/lib/elements/bfe_edit_record.py

```diff
@@ -36,7 +36,7 @@ def format_element(bfo, style):
     out = ""
 
     user_info = bfo.user_info
-    if user_can_edit_record_collection(user_info, bfo.recID):
+    if user_can_edit_record_collection(user_info, int(bfo.recID)):
```

Comment:
Shouldn't it be fixed somewhere in [`BibFormatObject`](https://github.com/inveniosoftware/invenio/blob/master/modules/bibformat/lib/bibformat_engine.py#L1870)?